week1_HW/todo-app/src/components/task.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+
+export default class Tasks extends Component {
```

Comment:
The component name `Tasks` here is also incorrect, or at least does not match the filename.